src/plugins/python/scripts/lib/lbm.py

```diff
@@ -57,28 +57,29 @@ class CMAP(CpyStruct("BYTE color[3];", True)):
   def parse(cls, dat):
     c = CMAP()
     sz = struct.calcsize(getattr(cls, '__fstr'))
-    for i in range(len(dat)/sz):
+    for i in range(int(len(dat)/sz)):
       c.unpack(dat[i*sz:i*sz+sz])
-      yield QColor(c.color[0],c.color[1],c.color[2]).rgb()
+      yield T.qt.QColor(c.color[0],c.color[1],c.color[2]).rgb()
 
 def uncomp(dat):
   i = 0
   ret = bytearray()
   while i < len(dat):
-    v = struct.unpack('b', dat[i])[0]
+    #v = struct.unpack('b', dat[i:i+1])[0]
+    v = (256-dat[i])*-1 if dat[i] > 127 else dat[i]
```

Comment:
This seems like a great improvement in readability in Python 3. o_O

Reply:
I take it you weren't serious?-) (the old way works too with the slice notation, I'll switch it back)